packaging/changelogs/deb.changelog

```diff
@@ -0,0 +1,6 @@
+pg-backup-api (0.1.0-1) unstable; urgency=low
```

Comment:
Again, probably missing something, but are the new changelogs and news files going to be committed to replace these at some point in the process?

Reply:
Yes - the idea would be to commit the changes to changelogs and news along with the other changes for the release. I'll update the release process to be more specific there.